src/components/Article.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Article = ({
+  title,
+  date,
+  text,
+}) => (
+  <>
+    <h1>{title}</h1>
+    <p>
+      {text}
+      {' '}
+      <span>{date}</span>
+    </p>
```

Comment:
```suggestion
    <h1>{title}</h1>
    <span>{date}</span>
    <p> {text}</p>
```
А почему ты не захотела сделать так, как в условии написано? Каждый параметр в свой тег

Reply:
мне показалось странным оставить спан "ничейным"

Reply:
Agreed. It is always good to follow task directions.